src/test/java/org/eclipse/yasson/defaultmapping/collections/ArrayTest.java

```diff
@@ -143,11 +144,14 @@ public void testUnmarshallMapWithArrayValue() {
     public void testArrayOfNulls() {
         String[] nulls = new String[2];
         String expected = "[null,null]";
-        assertEquals(expected, jsonb.toJson(nulls));
+        /*assertEquals(expected, jsonb.toJson(nulls));
```

Comment:
did you intend to leave this part of the test commented out?  If so, `String[] nulls` is no longer used.

Reply:
That is clearly not intentional, commented out during code fix and forgot about it. Good catch.